client.cfg

```diff
@@ -72,8 +72,8 @@ gcode:
   TURN_OFF_HEATERS
   M106 S0
   # clear pause_next_layer and pause_at_layer as preparation for next print
-  SET_GCODE_VARIABLE MACRO=SET_PRINT_STATS_INFO VARIABLE=pause_next_layer VALUE="{{'enable': False, 'call':"PAUSE"}}"
-  SET_GCODE_VARIABLE MACRO=SET_PRINT_STATS_INFO VARIABLE=pause_at_layer VALUE="{{'enable': False, 'layer': 0, 'call':"PAUSE"}}"
+  SET_PAUSE_NEXT_LAYER ENABLE=0
```

Comment:
Please change that to SET_PAUSE_NEXT_LAYER ENABLE=0 LAYER=0. I would prefer to get also the layer# reseted in an cancel print

Reply:
I assume you meant `SET_PAUSE_NEXT_LAYER`, right?

Reply:
I added a `LAYER=0` to the `SET_PAUSE_NEXT_LAYER` just below this line!